src/WorkflowClear.tsx

```diff
@@ -0,0 +1,15 @@
+import { useWorkflow } from './store'
+
+export const WorkflowClear = (): JSX.Element => {
+  const { clearNodes } = useWorkflow()
+
+  function clearWorkflow (): void {
+    clearNodes()
+  }
+
+  return (
+    <button className='btn btn-outline-danger' onClick={clearWorkflow}>
```

Comment:
The wrapping of clearNodes is not needed. You can directly use clearNodes as onClick value.
```suggestion
  return (
    <button className='btn btn-outline-danger' onClick={clearNodes}>
```